ModularContent/Sets/Set.php

```diff
@@ -96,14 +95,14 @@ public function get_template() {
 		if ( ! $this->post_id ) {
 			return [];
 		}
-		return PanelCollection::find_by_post_id( $this->post_id )->build_tree();
+		return apply_filters( 'panel_set_get_template', PanelCollection::find_by_post_id( $this->post_id )->build_tree(), $this->post_id );
 	}
 
 	/**
 	 * @return array The post types this Set can be applied to
 	 */
 	public function get_post_types() {
-		return get_post_meta( $this->post_id, self::META_KEY_POST_TYPES, false );
+		return apply_filters( 'panel_set_get_post_types', get_post_meta( $this->post_id, self::META_KEY_POST_TYPES, false ), $this->post_id );
```

Comment:
Similar to your approach above, I would set a var first then filter that. JIC there is some possible data or scope leakage.